README.md

```diff
@@ -351,14 +356,16 @@ Waggle Dance can be configured to use a prefix when resolving the names of datab
       name: primary
       remote-meta-store-uris: thrift://primaryLocalMetastore:9083
     federated-meta-stores:
-      - name: waggle_prod
+      - name: waggle_prod_
         remote-meta-store-uris: thrift://federatedProdMetastore:9083
 
-Using this example Waggle Dance will prefix all databases and will require the prefix to be present in queries in order to map to correct metastores. The query: `select * from waggle_prod_etldata` will effectively be this query: `select * from etldata` on the federated metastore. If a database is encountered that is not prefixed the primary metastore is used to resolve the database name. Any duplicate database name is made unique by prefixing it.
-
-Newly created databases are immediately accessible - no service restart is necessary.
+Note: When choosing a prefix ensure that it does not match the start of _any_ existing database names in your metastores. To illustrate the problem this would cause, 
+imagine you have a database in Metastore "A" named "my_database" and you configure another Metastore "B" with the prefix `my_`. Waggle Dance will register the prefix 
+and any requests for a database starting with `my_` will be routed to Metastore "B" even if they were intended to go to Metastore "A".
 
-Alternatively, Waggle Dance can be configured to use a static list of unprefixed databases in the configuration `waggle-dance-federations.yml`:`federated-meta-stores[n].mapped-databases`. Example configuration:
+In this mode any databases that are created while Waggle Dance is running will be automatically visible and will need to adhere to the naming rules described above 
+(e.g. not clash with the prefix). Alternatively, Waggle Dance can be configured to use a static list of unprefixed databases in the configuration 
+`waggle-dance-federations.yml`:`federated-meta-stores[n].mapped-databases`. Example configuration:
```

Comment:
@patduin if you look at line 380 below, is this correct? There is a name but no prefix, doesn't there still need to be a prefix for this to work? Maybe this is left over from when we allowed the name to be used as a prefix but decided to separate the two concepts?

Reply:
I've just tried it and it's correct. If there are no clashing database names, it works fine, but with name clashes, an error is thrown.

Reply:
Having looked at it, I think the current config examples are missing the `prefix` so I have added them.